infrastructure/terraform/ssm.tf

```diff
@@ -0,0 +1,556 @@
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "nexus_creds" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/NEXUS_CREDS"
+  description = "NEXUS_CREDS"
+  type        = "SecureString"
+  value       = "secret"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "nexus_host" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/NEXUS_HOST"
+  description = "NEXUS_HOST"
+  type        = "String"
+  value       = "ec2-52-201-239-45.compute-1.amazonaws.com"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "org_name" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/ORG_NAME"
+  description = "ORG_NAME"
+  type        = "String"
+  value       = "cpc-plus-conversion-tool"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "splunk_token" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/SPLUNK_TOKEN"
+  description = "SPLUNK_TOKEN"
+  type        = "SecureString"
+  value       = "secret"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "validation_url" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/VALIDATION_URL"
+  description = "VALIDATION_URL"
+  type        = "String"
+  value       = "https://preview.qpp.cms.gov/api/submissions/public/validate-submission"
```

Comment:
This will need to be changed for prod. From what i've been told, the dev-pre validation api was the only one that worked for testing purposes. This is a heads up for when we are deploying to prod.

Reply:
Plan is to set those manually - the TF will create the SSM entries but I'll need to update them manually because we don't want them checked into git. This is true for all of the "SecureString" settings as well as settings that differ in various environments. I am going to update the README